jupyter/trustyai/ubi9-python-3.9/test/test_notebook.ipynb

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
Upon reviewing the test notebook for the Ubi9, it appears to be distinct from the one I examined during my previous review. Could you please revert it?
It should look like the ubi8 -> https://github.com/dibryant/notebooks/blob/trustyai/jupyter/trustyai/ubi8-python-3.8/test/test_notebook.ipynb